mitmproxy-rs/src/process_info.rs

```diff
@@ -50,16 +49,9 @@ impl Process {
 /// *Availability: Windows, macOS*
 #[pyfunction]
 pub fn active_executables() -> PyResult<Vec<Process>> {
-    #[cfg(any(windows, target_os = "macos"))]
-    {
-        processes::active_executables()
-            .map(|p| p.into_iter().map(Process).collect())
-            .map_err(|e| pyo3::exceptions::PyRuntimeError::new_err(format!("{}", e)))
-    }
-    #[cfg(not(any(windows, target_os = "macos")))]
-    Err(pyo3::exceptions::PyNotImplementedError::new_err(
-        "active_executables is only available on Windows",
-    ))
+    processes::active_executables()
+        .map(|p| p.into_iter().map(Process).collect())
+        .map_err(|e| pyo3::exceptions::PyRuntimeError::new_err(format!("{}", e)))
```

Comment:
There are more OSes than just Windows, Linux and macOS. We should still do `#[cfg(any(windows, target_os = "macos", target_os = "linux"))]` for the FreeBSD users out there. :)

Reply:
yup, just updated that